Makefile

```diff
@@ -103,6 +104,11 @@ runservertests: $(BIN)/connectconformance $(BIN)/referenceserver $(BIN)/grpcserv
 runclienttests: $(BIN)/connectconformance $(BIN)/referenceclient $(BIN)/grpcclient
 	$(BIN)/connectconformance -v --conf ./testdata/reference-impls-config.yaml --mode client -- $(BIN)/referenceclient
 	$(BIN)/connectconformance -v --conf ./testdata/grpc-impls-config.yaml --mode client -- $(BIN)/grpcclient
+	$(MAKE) rungrpcweb
+
+.PHONY: rungrpcweb
+rungrpcweb: generate $(BIN)/connectconformance
+	cd grpcwebclient && npm run conformance
```

Comment:
Not sure of the best place to put the code. Didn't want it mixed in with the Go code, so lmk if you think there's a better place.